src/FormGenerator.js

```diff
@@ -0,0 +1,95 @@
+//Form creation tool
+module.exports.generateForm = function(input){
+  //String variable to hold final form render
+  var formStringHTML = '';
+
+  //For each data in input array
+  input.forEach(function(input){
```

Comment:
When preparing strings for a switch statement it would be a good idea to bring case to either upper ``` .toUpperCase() ``` or to lower ``` toLowerCase() ``` 
we wouldn't want to get a false match do to case alone.

a map array helper could be one route among many:
```javascript
let arrayRaw = ['Label', 'NAME', 'passWord'];

let arrayLower = arrayRaw.map(function(input){
	return input.toLowerCase();
});
// New array made by map array helper
console.log(arrayLower);
```

[toLowerCase reference](http://processingjs.org/reference/String_toLowerCase_/) 
[JavaScript Cookbook 1.3](https://books.google.com/books?id=kW0s1IrlbnIC&pg=PA5&lpg=PA5&dq=javascript+tolowercase+prior+to+processing+data&source=bl&ots=P2QNxk64CS&sig=oGe_CRZyibuC_mVtfAueEtTC3IA&hl=en&sa=X&ved=0ahUKEwif5r3BtIrXAhUHJCYKHct2DGUQ6AEIUjAH#v=onepage&q=javascript%20tolowercase%20prior%20to%20processing%20data&f=false)